app/views/feed/contentSelectorPreview.phtml

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+	<body>
+		<?php if ($this->fatalError != '') { ?>
+			<p class="warning"><?= $this->fatalError ?></p>
+		<?php } else { ?>
+			<?php if ($this->selectorSuccess === false) { ?>
+				<p class="warning"><?= _t('feedback.sub.feed.selector_preview.no_result') ?></p>
+				<hr>
+			<?php } ?>
+			<center>
+				<div>
+					<input type="radio" id="freshrss_rendered" name="freshrss_type" checked>
+					<label for="freshrss_rendered"><?= _t('sub.feed.selector_preview.show_rendered') ?></label>
+
+					<input type="radio" id="freshrss_raw" name="freshrss_type">
+					<label for="freshrss_raw"><?= _t('sub.feed.selector_preview.show_raw') ?></label>
+				</div>
+			</center>
+
+			<hr>
+
+			<div id="freshrss_rendered_view"> <?= $this->htmlContent ?></div>
+			<pre id="freshrss_raw_view" hidden> <?= htmlspecialchars($this->htmlContent) ?></pre>
+		<?php } ?>
+
+	<script>
+		var rendered_node = document.getElementById("freshrss_rendered");
```

Comment:
```suggestion
		const rendered_node = document.getElementById('freshrss_rendered');
```
And the same for following variables

Reply:
Hm, this must have been added more recently; I distinctly recall saying to use `let` and `const` instead of `var`. :-)